boards/common/nrf52/nrf52832/Makefile.dep

```diff
@@ -1,4 +1,4 @@
-ifneq (,$(filter gnrc_netdev_default,$(USEMODULE)))
+ifneq (,$(filter netdev_default,$(USEMODULE)))
```

Comment:
This is now bringing `nimble_netif` (which is only implemented for GNRC), even when using lwip. That's why the paho-mqtt is failing currently I think.

Reply:
Ahh so this one was correct to use `gnrc_netdev_default`

Reply:
And I though this was going to be a straight forward PR..

Reply:
I guess the test should actually be `tests_driver_gnrc_netdev_common` since it relies on gnrc to work.

Reply:
I reverted the pertinent changes, I think `tests_driver_gnrc_netdev_common` would be a better name here,
but I'm not sure if its worth the name change.